specification/dimensions/servicename.md

```diff
@@ -1,7 +1,8 @@
 # Service Name
 
-Service Name is an identifier assigned to a service by the provider. The Service Name is commonly used for cost
-reporting, analysis and allocation scenarios.
+A service represents the offering that was purchased through the provider (e.g., cloud or professional services). For cloud resources, the service is a higher-level classification of the resource type and may include multiple usage charges of different types (e.g., compute, storage, networking). The service is not a classification of the type of usage. Instead, it represents the offering you might find on a provider's website.
```

Comment:
cloud itself isn't the service though right? a Should the service examples be "Cloud Kubernetes, a SaaS relational database or a professional services and things like that?

Reply:
What if I flip them?
```suggestion
A service represents the offering that was purchased through the provider (e.g., professional or cloud services). For cloud resources, the service is a higher-level classification of the resource type and may include multiple usage charges of different types (e.g., compute, storage, networking). The service is not a classification of the type of usage. Instead, it represents the offering you might find on a provider's website.
```